commercial/test/model/LookupTest.scala

```diff
@@ -87,7 +87,7 @@ import scala.concurrent.duration._
   }
 
   "latestContentByKeyword" should "find content ordered reverse chronologically for an existing keyword" in {
-    val contents = contentsForKeyword("technology/apple")
+    val contents = contentsForKeyword("technology/apps")
```

Comment:
Why? 

The `technology/apple` had a (likely rare) occurrence where 2 of the most recent 4 articles where published at the exact same millisecond. Since there's no guaranteed order in this situation, Frontend's "test" and CAPIs order were different, causing the test to fail.

I chose technology/apps as the new tag, but literally any other tag likely won't have this problem.

<img width="1357" alt="image" src="https://user-images.githubusercontent.com/9575458/159486729-9f3a7cab-d53c-4ab2-921b-0edc3385a45c.png">


I do question whether this test is needed. Its goal is to test the function we call CAPI with, which should always ask for the 'latest' content. But really it ends up just testing whether CAPI correctly ordered its response on not -> which we should be able to assume it does, with added risks of edge cases like this breaking the test.

Should we just delete this entire test?

Reply:
I would say its still testing some elements of the Lookup class, although that class mostly seems like its just a wrapper for CAPI.

Maybe sort by ID instead of webPublicationDate?

Reply:
I think because CAPI has sorted by webPublicationDate (or there own internal publication date), in usual circumstances that's the only sorting metric we could use that would allow the test to pass the way it's intended to

Reply:
Maybe we could cc @guardian / digital-cms (if that's the right team?)